feature-libs/asm/components/customer-selection/customer-selection.component.spec.ts

```diff
@@ -129,6 +129,7 @@ describe('CustomerSelectionComponent', () => {
     component.ngOnInit();
     fixture.detectChanges();
 
+    // TODO: (CXSPA-153) Change div.spinner to use cx-dot-spinner component
```

Comment:
Is that todo still relevant?
Also, beware of breaking changes when changinf the UI.

Reply:
This was originally changed to use `<cx-dot-spinner>` but was reverted for breaking changes.  This todo is for the 6.0 migration.

Reply:
As discussed we can create the follow up ticket for 6.0 already and refer to it in the comment instead of CXSPA-153.

Reply:
Todo ticket updated to task CXSPA-1291